runbooks/OutdatedVirtualMachineInstanceWorkloads.md

```diff
@@ -0,0 +1,61 @@
+# OutdatedVirtualMachineInstanceWorkloads
+
+## Meaning
+
+There are VMIs running in outdated virt-launcher pods 24 hours after the KubeVirt control plane update has completed.
+
+## Impact
+
+VMIs that have not been updated to run in the most recent virt-launcher pod may not have access to new KubeVirt features and will not have received any security fixes associated with the virt-launcher pod update.
+
+## Diagnosis
+
+You can identify the VMIs that are out-of-date by using the `kubevirt.io/outdatedLauncherImage` label as a label selector when listing VMIs. Below is an example of a command that will list all out-of-date VMIs across all namespaces within the cluster.
+
+```bash
+kubectl get vmi -l “kubevirt.io/outdatedLauncherImage” --all-namespaces
+```
+
+## Mitigation
+
+### Check if automated workload updates are enabled
+
+Check the KubeVirt CR used to install KubeVirt to see if the `workloadUpdateStrategy` is configured on the KubeVirt CR's spec.
+
+Below is an example with automated workload updates enabled. With this configuration, workloads which are live migratable will always be live migrated and non live migratable workloads will be evicted which causes a restart when `RunStrategy: Always` is used on the corresponding VM definition
+
+```
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  name: kubevirt
+  namespace: kubevirt
+spec:
+  imagePullPolicy: IfNotPresent
+  workloadUpdateStrategy:
+    workloadUpdateMethods:
+      - LiveMigrate
+      - Evict
+    batchEvictSize: 10
+    batchEvictInterval: "1m"
+```
+
+If automated workload updates are not enabled, consider enabling them. This will cause the KubeVirt control place to automatically update the VMI workloads using the methods defined in the `workloadUpdateMethods` field.
```

Comment:
@davidvossel  typo, place instead of plane.

Reply:
fixed!